public/pages/CreateMonitor/containers/CreateMonitor/utils/formikToMonitor.js

```diff
@@ -267,46 +274,35 @@ export function formikToGraphQuery(values) {
 }
 
 export function formikToDocLevelInput(values) {
-  let description = FORMIK_INITIAL_VALUES.description;
-  let indices = formikToIndices(values);
-  let queries = _.get(values, 'queries', FORMIK_INITIAL_VALUES.queries);
-  switch (values.searchType) {
-    case SEARCH_TYPE.GRAPH:
-      description = values.description;
-      queries = queries.map((query) => {
-        const formikToQuery = DOC_LEVEL_QUERY_MAP[query.operator].query(query);
-        return {
-          id: query.id,
-          name: query.queryName,
-          query: formikToQuery,
-          tags: query.tags,
-        };
-      });
-      break;
-    case SEARCH_TYPE.QUERY:
-      let query = _.get(values, 'query', '');
-      try {
-        query = JSON.parse(query);
-        description = _.get(query, 'description', description);
```

Comment:
The changes below always set `description: FORMIK_INITIAL_VALUES.description`. Was there a reason to move away from this switch block?